docs/cli.md

```diff
@@ -125,6 +125,64 @@ Usage:
       --validate=false: If true, use a schema to validate the input before sending it
 ```
 
+### run-container
+Easily run one or more replicas of an image.
+
+Creates a replica controller running the specified image, with one or more replicas.
+
+Examples:
+```sh
+  $ kubectl run-container nginx --image=dockerfile/nginx
+  <starts a single instance of nginx>
+
+  $ kubectl run-container nginx --image=dockerfile/nginx --replicas=5
+  <starts a replicated instance of nginx>
+  
+  $ kubectl run-container nginx --image=dockerfile/nginx --dry-run
+  <just print the corresponding API objects, don't actually send them to the apiserver>
+
+Usage: 
+  kubectl run-container <name> --image=<image> [--command=<some command here>] [--replicas=replicas] [--dry-run=<bool>] [flags]
```

Comment:
Thanks for removing command. Now we need to update the doc. (Yes, auto-generation would be cool.)